src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ThemeControllerTests.cs

```diff
@@ -19,10 +19,10 @@ public class ThemeControllerTests
         [Fact]
         public async Task GetThemes()
         {
-            var publicationService = new Mock<IPublicationService>(MockBehavior.Strict);
+            var publicationService = new Mock<IThemeService>(MockBehavior.Strict);
```

Comment:
I think we should rename this themeService now

Reply:
Good catch! Wish the IDE refactoring would do this automatically!